From 7626f7a9b57e83aad48566028f2d5dab4ad8a0f3 Mon Sep 17 00:00:00 2001
Date: Tue, 1 Nov 2022 17:01:26 +0100
Subject: examples: fix plugin name on MINGW32/64 and CLANG32/64

* examples/findplugin.in:
   add plugin name on MINGW32/64 and CLANG32/64
---
diff --git a/examples/findplugin.in b/examples/findplugin.in
index 0656823..29560ad 100644
--- a/examples/findplugin.in
+++ b/examples/findplugin.in
@@ -39,6 +39,11 @@ if test "x$FP_OS" = xDarwin ; then FP_ISOSX=1; fi
 FP_OS=`uname | cut -d '_'  -f 1`
 if test "x$FP_OS" = xCYGWIN ; then FP_ISCYGWIN=1; fi

+FP_IS_OTHER_WIN=
+case $FP_OS in
+  MINGW* ) FP_IS_OTHER_WIN=1;;
+esac
+
 FP_PLUGINS="$topbuilddir/plugins"

 FP_PLUGIN_LIB=
@@ -50,6 +55,8 @@ if test "x$FP_ISMSVC" != x ; then
   FP_PLUGIN_LIB="${FP_NAME}.dll"
 elif test "x$FP_ISCYGWIN" != x ; then
   FP_PLUGIN_LIB="cyg${FP_NAME}.dll"
+elif test "x$FP_IS_OTHER_WIN" != x ; then
+  FP_PLUGIN_LIB="lib${FP_NAME}.dll"
 elif test "x$FP_ISOSX" != x ; then
   FP_PLUGIN_LIB="lib${FP_NAME}.so" # Should this include the version number in the name?
 else # Presumably some form on *nix"
--
2.36.1.windows.1

